src/modules/commander/state_machine_helper.h

```diff
@@ -58,6 +58,16 @@ typedef enum {
 
 } transition_result_t;
 
+enum class link_loss_actions_t
+{
+	DISABLED = 0,
+	AUTO_LOITER = 1,
+	AUTO_RTL = 2,
+	AUTO_LAND = 3,
+	AUTO_RECOVER = 4,
```

Comment:
It is from existing code, I just gave it a name based on existing param description.